HW9/Gnatenko_9_PyGame/gnatenko_pygame.py

```
@@ -0,0 +1,92 @@
+    no_attempts_left = False
+    won = False
+
+    while True:
+        for event in pygame.event.get(): # User did something
+            if event.type == pygame.QUIT: # If user clicked close
+                pygame.quit()
+                sys.exit()
+
+            if not no_attempts_left:
+
+                if event.type == pygame.KEYDOWN:
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)